lib/types/multipart.js

```diff
@@ -139,7 +140,8 @@ function Multipart (boy, cfg) {
         for (i = 0, len = parsed.length; i < len; ++i) {
           if (RE_NAME.test(parsed[i][0])) {
             fieldname = parsed[i][1]
-          } else if (RE_FILENAME.test(parsed[i][0])) {
+          }
+          if (RE_FILENAME.test(parsed[i][0])) {
```

Comment:
what was the intention behind removing the else? can something be both field and file?